_services/hubspot.html

```diff
@@ -64,7 +64,7 @@ <h2>Our Services</h2>
 </div>
 
 <div class="flex-row">
-  <img class="side-image" src="/img/hubspot-automated-sales-emails-en.png" />
+  <img class="side-image" src="/img/Create a New Pipeline in HubSpot.jpg" />
```

Comment:
Looks jarring to me that the image names are with spaces. I guess that is legal, but we could have some shorter names like `hubspot-automation`, `hubspot-pipeline` etc.

Also a lot of images are missing alt-attributes. Even though they are mostly for decoration they should have an empty `alt=""`.

Reply:
Fixed missing alt attributes in ea3064a

Reply:
Fixed image files with white spaces in 8e25a3dc1235579bf8ca2ea02baa7ead046880d8